app/Module/GoogleMapsModule.php

```diff
@@ -3147,19 +3160,18 @@ private function placesEdit() {
 		}
 
 		// Update placelocation STREETVIEW fields
-		// TODO: This ought to be a POST request, rather than a GET request
-		if ($action == 'update_sv_params' && Auth::isAdmin()) {
+		if ($action == 'update_sv_params' && Auth::isAdmin() && Filter::checkCsrf()) {
 			Database::prepare(
 				"UPDATE `##placelocation` SET sv_lati=?, sv_long=?, sv_bearing=?, sv_elevation=?, sv_zoom=? WHERE pl_id=?"
 			)->execute(array(
-				Filter::get('svlati'),
-				Filter::get('svlong'),
-				Filter::get('svbear'),
-				Filter::get('svelev'),
-				Filter::get('svzoom'),
+				(float) Filter::post('sv_latiText'),
+				(float) Filter::post('sv_longText'),
+				(float) Filter::post('sv_bearText'),
+				(float) Filter::post('sv_elevText'),
+				(float) Filter::post('sv_zoomText'),
 				$placeid,
 			));
-			$controller->addInlineJavascript('window.close();');
+			header('Location: ' . Filter::post('destination', null, 'index.php'));
```

Comment:
This change isn't working for me.  I get a "headers already sent" error, due to the earlier `$controller->pageHeader();`.

I've submitted a fix for this.